.jenkins/pytorch/multigpu-test.sh

```diff
@@ -45,4 +45,5 @@ time python test/run_test.py --verbose -i distributed/_shard/test_partial_tensor
 time python test/run_test.py --verbose -i distributed/_shard/test_replicated_tensor
 # Other tests
 time python test/run_test.py --verbose -i test_cuda_primary_ctx
+time python test/run_test.py --verbose -i test_optim -- -k optimizers_with_varying_tensors
```

Comment:
@pytorch/pytorch-dev-infra to make sure this is okay. The total time it would add to multigpu is about 13 seconds.